### PR TITLE
tpl notices and docblocks

### PR DESCRIPTION
**Pull request by mattwire**
Overview
----------------------------------------
Just tpl php notices and docblock

Before
----------------------------------------

After
----------------------------------------

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

File: templates/CRM/Activity/Selector/Selector.tpl [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <div class="crm-activity-selector-{$context}">
   <div class="crm-accordion-wrapper crm-search_filters-accordion">
     <div class="crm-accordion-header">
-    {ts}Filter by Activity{/ts}</a>
```
Comment (mattwire) [inline templates/CRM/Activity/Selector/Selector.tpl:13]: There is no opening "<a>"

File: templates/CRM/Activity/Form/Selector.tpl [hunk 2]
```diff
@@ -67,7 +67,7 @@
     <td>
     {if $row.mailingId}
       <a href="{$row.mailingId}" title="{ts}View Mailing Report{/ts}">{$row.recipients}</a>
-    {elseif $row.recipients}
+    {elseif !empty($row.recipients)}
```
Comment (seamuslee001) [inline templates/CRM/Activity/Form/Selector.tpl:70]: I think this one might be an issue with the hardening @eileenmcnaughton is doing.

Comment (eileenmcnaughton) [reply templates/CRM/Activity/Form/Selector.tpl:70]: yeah - you can't add empty to tpls (or isset) - you need to make sure it is assigned or you are just moving the problem down the road. I've been taking out the empties

Comment (mattwire) [reply templates/CRM/Activity/Form/Selector.tpl:70]: Ok, I thought it was only the isset that had to go

**Comment by mattwire** [comment]
test this please

**Pull request merged**